translations/pt-br/CONTRIBUTING.md

```diff
@@ -0,0 +1,40 @@
+# Contribuindo
+
+Olá! Obrigado por querer contribuir com este projeto! Nós somos gratos aos nossos contribuidores.
+
+Todos os contribuidores concordam com o [Código de Conduta](./CODEOFCONDUCT.md).
```

Comment:
Probably should link to the PT-BR version here.

Reply:
Ah, wait. I see now that it does since you kept the same filenames. Excellent. 😄

Reply:
Yeah, I did it to follow a standard.